tools-adoption.md

```diff
@@ -0,0 +1,23 @@
+# Tools across our projects
+
+This page is supposed to track the adoption of the tools that want to use across our projects.
+
+## Goals
+
+- Migrate from Travis CI to GitHub Actions
+- Migrate GitHub wiki documentation to Docusaurus
+- Have a deployable version of each app. E.g.: Heroku or GitHub Pages or mobile apk
+
+## Tracking
+
+| Project | GitHub Actions | Docusaurus | Heroku | GitHub Pages |
+|-|:-:|:-:|:-:|:-:|
+| anitab-org.github.io | ✅ | ❌ | N/A | ✅ |
+| mentorship-backend | ✅ | ❌ | ✅ | N/A |
+| mentorship-android | ❌ | ❌ | N/A | ✅ |
+| stem-diverse-tv | ❌ | ❌ | ✅ | N/A |
+| stem-diverse-tv-cms | ❌ | ❌ | ❌ | N/A |
```

Comment:
@isabelcosta [Stem-diverse-tv-cms](https://anitab-org.github.io/stem-diverse-tv-cms/) is deployed to github pages as well.